res/values-v31/themes.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think I'd prefer to drop this file. These look like the Monet theme but worse on my device.

Reply:
I love you android (sarcasm)